tests/cloud_tests/platforms/lxd/image.py

```diff
@@ -76,19 +76,22 @@ def properties(self):
         }
 
     def export_image(self, output_dir):
-        """Export image from lxd image store to (split) tarball on disk.
+        """Export image from lxd image store to disk
 
-        @param output_dir: dir to store tarballs in
-        @return_value: tuple of path to metadata tarball and rootfs tarball
+        @param output_dir: dir to store the exported image in
+        @return_value: tuple of path to metadata tarball and rootfs
         """
         # pylxd's image export feature doesn't do split exports, so use cmdline
         subp.subp(['lxc', 'image', 'export', self.pylxd_image.fingerprint,
                   output_dir], capture=True)
-        tarballs = [p for p in os.listdir(output_dir) if p.endswith('tar.xz')]
```

Comment:
This precludes the previous tarball format.  Also, the .tar.xz wont handle .gz...

Reply:
It's not just an issue with the compression format: when LXD exports to a single .tar.gz the file also contains the rootfs, while the .tar.xz export normally has the rootfs in a separate squashfs.

But I think that even this is guessing: we could have a "split" export with a .tar.gz metadata file I think, depending on how the image is imported. I don't think there's a way to have LXD convert between image formats, but maybe there's a way to know the image type in advance. I'm looking/asking.

Reply:
I think if we collect what matches the fingerprint, then we can determine if we're split (meta + root (in whatever form)) or single tarball.

I suspect we've never supported single tarball, so instead of making that work, we can just raise an exception (unhandled image export format?)

Reply:
+1